### ISSUE TITLE
Detect changes in imported files

### ISSUE THREAD

**Issue by tindzk** [ShaggyYeti/sbt-sass#18]
When changing an imported `.css` file, I need to modify the main `.scss` file as well, before I can use `sbt assets` again. Would it be possible to scan all imports and detect changes in these, too?

**Comment by guersam** [comment]
It seems to be due to the incremental task API of sbt-web. Possible solutions are:
1. Add an option to turn on/off incremental compilation
2. Enhance incremental compilation with custom hasher which understands the import chain

Solution 1 must be easier to implement at this time. Another option is to let Sass/Compass handle it, but I'm new to Sass so that I don't know if they support incremental compilation.

**Comment by tindzk** [comment]
Yes, Sass supports incremental compilation with the `--watch` parameter (see http://sass-lang.com/documentation/file.SASS_REFERENCE.html). I'd also prefer 1) because we wouldn't need to reimplement a subset of the Sass parser.
